web/src/routes/Dashboard/DashboardListProject.js

```diff
@@ -99,9 +99,8 @@ function DashboardListProject({
               {project.members.map(
                 (member) =>
                   member && (
-                    <div className="dashboard-list-project-member-wrapper">
+                    <div key={member.address} className="dashboard-list-project-member-wrapper">
```

Comment:
@pegahvaezi  ... when we use `.map` ... the very top-level component, be it a 'div' or a custom component like <Avatar> has to have a `key`. The `key` must be unique so that React can track which node is which in a flat list of items so that it can appropriately and and remove them when the data changes. So when you added the `div` wrapper recently, it would be necessary to move the `key` up to that level like I've done here.